nav/dataset/dataset_pds3_cassini_iss.py

```
@@ -97,37 +139,64 @@ def _extract_img_number(img_name: str) -> int:
 
-    _MIN_1xxx_VOL = 1001
-    _MAX_1xxx_VOL = 1009
-    _MIN_2xxx_VOL = 2001
-    _MAX_2xxx_VOL = 2116
+        if arguments is None or arguments.camera is None:
+            return True
+        if arguments.camera.lower() == 'nac':
+            return img_name[0] == 'N'
+        if arguments.camera.lower() == 'wac':
+            return img_name[0] == 'W'
+        raise ValueError(f'Invalid camera type "{arguments.camera}"')
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)